### ISSUE TITLE
King Phisher 1.15.0 fails to build pipenv 

### ISSUE THREAD

**Issue by True-Demon** [securestate/king-phisher#415]
## Issue Description
Using the king-phisher installer script, the install fails to build the pipenv properly, causing dependency issues and possibly further issues with the server. This appears to only be an issue with king-phisher v1.15.0+ (including 1.16.0-beta)

## Reproduction Steps
Run `wget -q https://github.com/securestate/king-phisher/raw/master/tools/install.sh && sudo bash ./install.sh` on a clean kali-light-amd64 system.

## Environment Details

Host OS: Linux kingphisher1 5.2.0-kali2-amd64 #1 SMP Debian 5.2.9-2kali1 (2019-08-22) x86_64 GNU/Linux
King Phisher Version: Version: 1.15.0 (rev: 0be34a20b0f3)

## Error Details / Stack Trace
```
root@kingphisher1:/opt# wget -q https://github.com/securestate/king-phisher/raw/master/tools/install.sh && \
> sudo bash ./install.sh
INFO: Linux version detected as Kali
Install and use PostgreSQL? (Highly recommended and required for upgrading) [Y/n] y
INFO: Will install and configure PostgreSQL for the server
INFO: Attempting to update apt-get cache package information
Get:1 http://kali.download/kali kali-rolling InRelease [30.5 kB]
Get:2 http://kali.download/kali kali-rolling/main amd64 Packages [16.5 MB]
Get:3 http://kali.download/kali kali-rolling/non-free amd64 Packages [191 kB]
Get:4 http://kali.download/kali kali-rolling/contrib amd64 Packages [104 kB]
Fetched 16.9 MB in 3s (6,565 kB/s)                     
Reading package lists... Done
INFO: Downloading and installing the King Phisher server to /opt/king-phisher
INFO: Successfully cloned the git repo
INFO: Synchronizing Kali OS dependencies
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libfreetype6-dev is already the newest version (2.10.1-2).
pkg-config is already the newest version (0.29-6).
python3-dev is already the newest version (3.7.5-1).
0 upgraded, 0 newly installed, 0 to remove and 129 not upgraded.
pip 19.3.1 from /usr/local/lib/python3.7/dist-packages/pip (python 3.7)
Reading package lists... Done
Building dependency tree       
Reading state information... Done
gir1.2-gtk-3.0 is already the newest version (3.24.12-1).
gir1.2-gtksource-3.0 is already the newest version (3.24.11-2).
libgeos++-dev is already the newest version (3.8.0-1).
libgirepository1.0-dev is already the newest version (1.62.0-2).
libgtk-3-dev is already the newest version (3.24.12-1).
libpq-dev is already the newest version (12.1-1).
python3-gi is already the newest version (3.34.0-3).
python3-gi-cairo is already the newest version (3.34.0-3).
The following packages will be upgraded:
  python3-cairo
1 upgraded, 0 newly installed, 0 to remove and 128 not upgraded.
Need to get 71.9 kB of archives.
After this operation, 8,192 B of additional disk space will be used.
Get:1 http://kali.download/kali kali-rolling/main amd64 python3-cairo amd64 1.16.2-2 [71.9 kB]
Fetched 71.9 kB in 1s (126 kB/s)   
apt-listchanges: Reading changelogs...
(Reading database ... 85686 files and directories currently installed.)
Preparing to unpack .../python3-cairo_1.16.2-2_amd64.deb ...
Unpacking python3-cairo:amd64 (1.16.2-2) over (1.16.2-1+b1) ...
Setting up python3-cairo:amd64 (1.16.2-2) ...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
gir1.2-vte-2.91 is already the newest version (0.58.2-1).
0 upgraded, 0 newly installed, 0 to remove and 128 not upgraded.
INFO: Successfully installed gir1.2-webkit2-4.0 with apt-get
INFO: Synchronizing Python package dependencies from PyPi
pip 19.3.1 from /usr/local/lib/python3.7/dist-packages/pip (python 3.7)
Requirement already up-to-date: pip in /usr/local/lib/python3.7/dist-packages (19.3.1)
Requirement already up-to-date: setuptools in /usr/local/lib/python3.7/dist-packages (41.6.0)
Requirement already up-to-date: six in /usr/local/lib/python3.7/dist-packages (1.13.0)
Requirement already satisfied: pipenv in /usr/local/lib/python3.7/dist-packages (2018.11.26)
Requirement already satisfied: virtualenv in /usr/local/lib/python3.7/dist-packages (from pipenv) (16.7.7)
Requirement already satisfied: setuptools>=36.2.1 in /usr/local/lib/python3.7/dist-packages (from pipenv) (41.6.0)
Requirement already satisfied: pip>=9.0.1 in /usr/local/lib/python3.7/dist-packages (from pipenv) (19.3.1)
Requirement already satisfied: virtualenv-clone>=0.2.5 in /usr/local/lib/python3.7/dist-packages (from pipenv) (0.5.3)
Requirement already satisfied: certifi in /usr/lib/python3/dist-packages (from pipenv) (2018.8.24)
INFO: Synchronizing King Phisher's pipenv environment
Creating a virtualenv for this project…
Pipfile: /opt/king-phisher/Pipfile
Using /usr/bin/python3.7m (3.7.5) to create virtualenv…
Making site-packages available…
⠙ Creating virtual environment...Already using interpreter /usr/bin/python3.7m
Using base prefix '/usr'
New python executable in /opt/king-phisher/.venv/bin/python3.7m
Also creating executable in /opt/king-phisher/.venv/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /usr/bin/python3.7m

✔ Successfully created virtual environment! 
Virtualenv location: /opt/king-phisher/.venv
Virtualenv already exists!
Removing existing virtualenv…
Creating a virtualenv for this project…
Pipfile: /opt/king-phisher/Pipfile
Using /usr/bin/python3.7m (3.7.5) to create virtualenv…
⠦ Creating virtual environment...Already using interpreter /usr/bin/python3.7m
Using base prefix '/usr'
New python executable in /opt/king-phisher/.venv/bin/python3.7m
Also creating executable in /opt/king-phisher/.venv/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /usr/bin/python3.7m

✔ Successfully created virtual environment! 
Virtualenv location: /opt/king-phisher/.venv
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
⠧ Locking..✔ Success! 
Locking [packages] dependencies…
⠦ Locking..✔ Success! 
UIpdated Pipfile.lock (443c1c)!
An error occurred while installing https://github.com/matplotlib/basemap/archive/v1.2.1rel.tar.gz#egg=basemap! Will try again.
An error occurred while installing bcrypt==3.1.7 --hash=sha256:0258f143f3de96b7c14f762c770f5fc56ccd72f8a1857a451c1cd9a655d9ac89 --hash=sha256:0b0069c752ec14172c5f78208f1863d7ad6755a6fae6fe76ec2c80d13be41e42 --hash=sha256:19a4b72a6ae5bb467fea018b825f0a7d917789bcfe893e53f15c92805d187294 --hash=sha256:5432dd7b34107ae8ed6c10a71b4397f1c853bd39a4d6ffa7e35f40584cffd161 --hash=sha256:6305557019906466fc42dbc53b46da004e72fd7a551c044a827e572c82191752 --hash=sha256:69361315039878c0680be456640f8705d76cb4a3a3fe1e057e0f261b74be4b31 --hash=sha256:6fe49a60b25b584e2f4ef175b29d3a83ba63b3a4df1b4c0605b826668d1b6be5 --hash=sha256:74a015102e877d0ccd02cdeaa18b32aa7273746914a6c5d0456dd442cb65b99c --hash=sha256:763669a367869786bb4c8fcf731f4175775a5b43f070f50f46f0b59da45375d0 --hash=sha256:8b10acde4e1919d6015e1df86d4c217d3b5b01bb7744c36113ea43d529e1c3de --hash=sha256:9fe92406c857409b70a38729dbdf6578caf9228de0aef5bc44f859ffe971a39e --hash=sha256:a190f2a5dbbdbff4b74e3103cef44344bc30e61255beb27310e2aec407766052 --hash=sha256:a595c12c618119255c90deb4b046e1ca3bcfad64667c43d1166f2b04bc72db09 --hash=sha256:c9457fa5c121e94a58d6505cadca8bed1c64444b83b3204928a866ca2e599105 --hash=sha256:cb93f6b2ab0f6853550b74e051d297c27a638719753eb9ff66d1e4072be67133 --hash=sha256:ce4e4f0deb51d38b1611a27f330426154f2980e66582dc5f438aad38b5f24fc1 --hash=sha256:d7bdc26475679dd073ba0ed2766445bb5b20ca4793ca0db32b399dccc6bc84b7 --hash=sha256:ff032765bb8716d9387fd5376d987a937254b0619eff0972779515b5c98820bc! Will try again.
An error occurred while installing cffi==1.13.2 --hash=sha256:0b49274afc941c626b605fb59b59c3485c17dc776dc3cc7cc14aca74cc19cc42 --hash=sha256:0e3ea92942cb1168e38c05c1d56b0527ce31f1a370f6117f1d490b8dcd6b3a04 --hash=sha256:135f69aecbf4517d5b3d6429207b2dff49c876be724ac0c8bf8e1ea99df3d7e5 --hash=sha256:19db0cdd6e516f13329cba4903368bff9bb5a9331d3410b1b448daaadc495e54 --hash=sha256:2781e9ad0e9d47173c0093321bb5435a9dfae0ed6a762aabafa13108f5f7b2ba --hash=sha256:291f7c42e21d72144bb1c1b2e825ec60f46d0a7468f5346841860454c7aa8f57 --hash=sha256:2c5e309ec482556397cb21ede0350c5e82f0eb2621de04b2633588d118da4396 --hash=sha256:2e9c80a8c3344a92cb04661115898a9129c074f7ab82011ef4b612f645939f12 --hash=sha256:32a262e2b90ffcfdd97c7a5e24a6012a43c61f1f5a57789ad80af1d26c6acd97 --hash=sha256:3c9fff570f13480b201e9ab69453108f6d98244a7f495e91b6c654a47486ba43 --hash=sha256:415bdc7ca8c1c634a6d7163d43fb0ea885a07e9618a64bda407e04b04333b7db --hash=sha256:42194f54c11abc8583417a7cf4eaff544ce0de8187abaf5d29029c91b1725ad3 --hash=sha256:4424e42199e86b21fc4db83bd76909a6fc2a2aefb352cb5414833c030f6ed71b --hash=sha256:4a43c91840bda5f55249413037b7a9b79c90b1184ed504883b72c4df70778579 --hash=sha256:599a1e8ff057ac530c9ad1778293c665cb81a791421f46922d80a86473c13346 --hash=sha256:5c4fae4e9cdd18c82ba3a134be256e98dc0596af1e7285a3d2602c97dcfa5159 --hash=sha256:5ecfa867dea6fabe2a58f03ac9186ea64da1386af2159196da51c4904e11d652 --hash=sha256:62f2578358d3a92e4ab2d830cd1c2049c9c0d0e6d3c58322993cc341bdeac22e --hash=sha256:6471a82d5abea994e38d2c2abc77164b4f7fbaaf80261cb98394d5793f11b12a --hash=sha256:6d4f18483d040e18546108eb13b1dfa1000a089bcf8529e30346116ea6240506 --hash=sha256:71a608532ab3bd26223c8d841dde43f3516aa5d2bf37b50ac410bb5e99053e8f --hash=sha256:74a1d8c85fb6ff0b30fbfa8ad0ac23cd601a138f7509dc617ebc65ef305bb98d --hash=sha256:7b93a885bb13073afb0aa73ad82059a4c41f4b7d8eb8368980448b52d4c7dc2c --hash=sha256:7d4751da932caaec419d514eaa4215eaf14b612cff66398dd51129ac22680b20 --hash=sha256:7f627141a26b551bdebbc4855c1157feeef18241b4b8366ed22a5c7d672ef858 --hash=sha256:8169cf44dd8f9071b2b9248c35fc35e8677451c52f795daa2bb4643f32a540bc --hash=sha256:aa00d66c0fab27373ae44ae26a66a9e43ff2a678bf63a9c7c1a9a4d61172827a --hash=sha256:ccb032fda0873254380aa2bfad2582aedc2959186cce61e3a17abc1a55ff89c3 --hash=sha256:d754f39e0d1603b5b24a7f8484b22d2904fa551fe865fd0d4c3332f078d20d4e --hash=sha256:d75c461e20e29afc0aee7172a0950157c704ff0dd51613506bd7d82b718e7410 --hash=sha256:dcd65317dd15bc0451f3e01c80da2216a31916bdcffd6221ca1202d96584aa25 --hash=sha256:e570d3ab32e2c2861c4ebe6ffcad6a8abf9347432a37608fe1fbd157b3f0036b --hash=sha256:fd43a88e045cf992ed09fa724b5315b790525f2676883a6ea64e3263bae6549d! Will try again.
An error occurred while installing cryptography==2.7 --hash=sha256:24b61e5fcb506424d3ec4e18bca995833839bf13c59fc43e530e488f28d46b8c --hash=sha256:25dd1581a183e9e7a806fe0543f485103232f940fcfc301db65e630512cce643 --hash=sha256:3452bba7c21c69f2df772762be0066c7ed5dc65df494a1d53a58b683a83e1216 --hash=sha256:41a0be220dd1ed9e998f5891948306eb8c812b512dc398e5a01846d855050799 --hash=sha256:5751d8a11b956fbfa314f6553d186b94aa70fdb03d8a4d4f1c82dcacf0cbe28a --hash=sha256:5f61c7d749048fa6e3322258b4263463bfccefecb0dd731b6561cb617a1d9bb9 --hash=sha256:72e24c521fa2106f19623a3851e9f89ddfdeb9ac63871c7643790f872a305dfc --hash=sha256:7b97ae6ef5cba2e3bb14256625423413d5ce8d1abb91d4f29b6d1a081da765f8 --hash=sha256:961e886d8a3590fd2c723cf07be14e2a91cf53c25f02435c04d39e90780e3b53 --hash=sha256:96d8473848e984184b6728e2c9d391482008646276c3ff084a1bd89e15ff53a1 --hash=sha256:ae536da50c7ad1e002c3eee101871d93abdc90d9c5f651818450a0d3af718609 --hash=sha256:b0db0cecf396033abb4a93c95d1602f268b3a68bb0a9cc06a7cff587bb9a7292 --hash=sha256:cfee9164954c186b191b91d4193989ca994703b2fff406f71cf454a2d3c7327e --hash=sha256:e6347742ac8f35ded4a46ff835c60e68c22a536a8ae5c4422966d06946b6d4c6 --hash=sha256:f27d93f0139a3c056172ebb5d4f9056e770fdf0206c2f422ff2ebbad142e09ed --hash=sha256:f57b76e46a58b63d1c6375017f4564a28f19a5ca912691fd2e4261b3414b618d! Will try again.
An error occurred while installing cycler==0.10.0 --hash=sha256:1d8a5ae1ff6c5cf9b93e8811e581232ad8920aeec647c37316ceac982b08cb2d --hash=sha256:cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8! Will try again.
An error occurred while installing email-validator==1.0.4 --hash=sha256:79966e318d6d68fed359c90f8f19d242bcc178b724011f1c07145bd093da6cc7! Will try again.
An error occurred while installing geoip2==2.9.0 --hash=sha256:a37ddac2d200ffb97c736da8b8ba9d5d8dc47da6ec0f162a461b681ecac53a14 --hash=sha256:f7ffe9d258e71a42cf622ce6350d976de1d0312b9f2fbce3975c7d838b57ecf0! Will try again.
An error occurred while installing graphene==2.1.8 --hash=sha256:09165f03e1591b76bf57b133482db9be6dac72c74b0a628d3c93182af9c5a896 --hash=sha256:2cbe6d4ef15cfc7b7805e0760a0e5b80747161ce1b0f990dfdc0d2cf497c12f9! Will try again.
An error occurred while installing graphene-sqlalchemy==2.1.2 --hash=sha256:107cba4248e69de95bd03cd50485b9d998fac449ece5ef9f44b8b7e09dac6db7 --hash=sha256:1524eace8823dc6e91d26942666db8706a1d877dfad4b47f3dec007312014e05! Will try again.
An error occurred while installing graphql-core==2.2.1 --hash=sha256:1488f2a5c2272dc9ba66e3042a6d1c30cea0db4c80bd1e911c6791ad6187d91b --hash=sha256:da64c472d720da4537a2e8de8ba859210b62841bd47a9be65ca35177f62fe0e4! Will try again.
An error occurred while installing graphql-relay==2.0.0 --hash=sha256:0e94201af4089e1f81f07d7bd8f84799768e39d70fa1ea16d1df505b46cc6335 --hash=sha256:75aa0758971e252964cb94068a4decd472d2a8295229f02189e3cbca1f10dbb5 --hash=sha256:7fa74661246e826ef939ee92e768f698df167a7617361ab399901eaebf80dce6! Will try again.
An error occurred while installing icalendar==4.0.3 --hash=sha256:07c2447a1d44cbb27c90b8c6a5c98e890cc1853c6223e2a52195cddec26c6356 --hash=sha256:83f7248b7485ddd29c7d69b706b21c441e34855d9c1d888939fd24aefdd9d19b! Will try again.
An error occurred while installing jinja2==2.10.1 --hash=sha256:065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013 --hash=sha256:14dd6caf1527abb21f08f86c784eac40853ba93edb79552aa1e4b8aef1b61c7b! Will try again.
An error occurred while installing matplotlib==2.2.4 --hash=sha256:029620799e581802961ac1dcff5cb5d3ee2f602e0db9c0f202a90495b37d2126 --hash=sha256:2308f67e085735ed580fcace652339cb517f059cdc9ee8a418c1b55746dbffcb --hash=sha256:280aebaec25575e35bf7d1b3ebb2d8ae7e839edb5a403f1a121b7271744b1ef9 --hash=sha256:295099acb5a8a1148d1b4693ad1a93479a20836cd8b7eb38183a98c84cdcb2f1 --hash=sha256:75d44c55eb87af653afc3d0a37ab62ab4784c752be0e7c96622713d88ed57e64 --hash=sha256:95d9d7c2d7f0c7a4317acbcf1a81efa0a2ce5cb5ddfad606ae4c25a783431f0a --hash=sha256:9703ffc3e7e369f3ab31d0032719710876cb341eb618e1a8a54447e1946a9f0a --hash=sha256:9ff80541d5676207c6e829632b28e22d9875ecaae54eab7a7f8fd82a6552e5e9 --hash=sha256:a6a04ebd81b3183e7882c9047a9514b7f547b2bae5e4f61a02eaaa6b446bde54 --hash=sha256:b22b0d3b8d8f769c6ac559f6761878d660bd23d67b36430f07161caf1505c29c --hash=sha256:b464d598e36e13f7d798443805f2ba6b4af3d26fc1652c51c77a7847cf665813 --hash=sha256:c0fa162920185d5d74e6fdf52c1f8cca0fbf897025a9dd81e030cf08a915865a --hash=sha256:c452b7aff0a9e4612670a4590e6efc30929dad620a121d423c8f3d0bd93715e2 --hash=sha256:c90fc796e97815ea3bbbdea63c1e4edf75336361a49b945fdbc2aff1c76008c6 --hash=sha256:cc1d376963ea9c97338582f3f9d64757c51e71cf2655efe363a3f2414d84aac2 --hash=sha256:d3f5dfaa345539599308bd83826db242e424e3f4e9657952f8738ce1b5b90e8a --hash=sha256:d9e80ba0ffdb0daacaf49e561474d5c5c153d6db853478cf90c8cba5ed8b72b1 --hash=sha256:daac44fc77cf36ff01953e2acc57a843fb1f6572eb5bf0af10a2930fa7407715 --hash=sha256:de43c85335d71094a254e8538719752e30db3305005dae8dcb3097b72587ed07 --hash=sha256:e4621af28a2444f93b5b6d3d60f54767df8ac6daa510a98f68c34377cb474869 --hash=sha256:f3755a52aae7fb640f5f57b7b63eb5d65688c84931d7833dbc7d03959cd4f8ce --hash=sha256:f99c43df8ed2b9d1c95a042f3cacf017f9690092feba0b4292eaa6713f92de97! Will try again.
An error occurred while installing paramiko==2.6.0 --hash=sha256:99f0179bdc176281d21961a003ffdb2ec369daac1a1007241f53374e376576cf --hash=sha256:f4b2edfa0d226b70bd4ca31ea7e389325990283da23465d572ed1f70a7583041! Will try again.
An error occurred while installing promise==2.2.1 --hash=sha256:2ebbfc10b7abf6354403ed785fe4f04b9dfd421eb1a474ac8d187022228332af --hash=sha256:348f5f6c3edd4fd47c9cd65aed03ac1b31136d375aa63871a57d3e444c85655c! Will try again.
An error occurred while installing pygobject==3.32.2 --hash=sha256:fa82746df059b9e0dcf473e0a9ff0693aa5e064f1ca1e2c46b5ac815bceda1ad! Will try again.
An error occurred while installing pynacl==1.3.0 --hash=sha256:05c26f93964373fc0abe332676cb6735f0ecad27711035b9472751faa8521255 --hash=sha256:0c6100edd16fefd1557da078c7a31e7b7d7a52ce39fdca2bec29d4f7b6e7600c --hash=sha256:0d0a8171a68edf51add1e73d2159c4bc19fc0718e79dec51166e940856c2f28e --hash=sha256:1c780712b206317a746ace34c209b8c29dbfd841dfbc02aa27f2084dd3db77ae --hash=sha256:2424c8b9f41aa65bbdbd7a64e73a7450ebb4aa9ddedc6a081e7afcc4c97f7621 --hash=sha256:2d23c04e8d709444220557ae48ed01f3f1086439f12dbf11976e849a4926db56 --hash=sha256:30f36a9c70450c7878053fa1344aca0145fd47d845270b43a7ee9192a051bf39 --hash=sha256:37aa336a317209f1bb099ad177fef0da45be36a2aa664507c5d72015f956c310 --hash=sha256:4943decfc5b905748f0756fdd99d4f9498d7064815c4cf3643820c9028b711d1 --hash=sha256:53126cd91356342dcae7e209f840212a58dcf1177ad52c1d938d428eebc9fee5 --hash=sha256:57ef38a65056e7800859e5ba9e6091053cd06e1038983016effaffe0efcd594a --hash=sha256:5bd61e9b44c543016ce1f6aef48606280e45f892a928ca7068fba30021e9b786 --hash=sha256:6482d3017a0c0327a49dddc8bd1074cc730d45db2ccb09c3bac1f8f32d1eb61b --hash=sha256:7d3ce02c0784b7cbcc771a2da6ea51f87e8716004512493a2b69016326301c3b --hash=sha256:a14e499c0f5955dcc3991f785f3f8e2130ed504fa3a7f44009ff458ad6bdd17f --hash=sha256:a39f54ccbcd2757d1d63b0ec00a00980c0b382c62865b61a505163943624ab20 --hash=sha256:aabb0c5232910a20eec8563503c153a8e78bbf5459490c49ab31f6adf3f3a415 --hash=sha256:bd4ecb473a96ad0f90c20acba4f0bf0df91a4e03a1f4dd6a4bdc9ca75aa3a715 --hash=sha256:bf459128feb543cfca16a95f8da31e2e65e4c5257d2f3dfa8c0c1031139c9c92 --hash=sha256:e2da3c13307eac601f3de04887624939aca8ee3c9488a0bb0eca4fb9401fc6b1 --hash=sha256:f67814c38162f4deb31f68d590771a29d5ae3b1bd64b75cf232308e5c74777e0! Will try again.
An error occurred while installing python-dateutil==2.8.1 --hash=sha256:73ebfe9dbf22e832286dafa60473e4cd239f8592f699aa5adaf10050e6e1823c --hash=sha256:75bb3f31ea686f1197762692a9ee6a7550b59fc6ca3a1f4b5d7e32fb98e2da2a! Will try again.
An error occurred while installing requests==2.22.0 --hash=sha256:11e007a8a2aa0323f5a921e9e6a2d7e4e67d9877e85773fba9ba6419025cbeb4 --hash=sha256:9cf5292fcd0f598c671cfc1e0d7d1a7f13bb8085e9a590f48c010551dc6c4b31! Will try again.
An error occurred while installing requests-file==1.4.3 --hash=sha256:75c175eed739270aec3c5279ffd74e6527dada275c5c0d76b5817e9c86bb7dea --hash=sha256:8f04aa6201bacda0567e7ac7f677f1499b0fc76b22140c54bc06edf1ba92e2fa! Will try again.
An error occurred while installing rule-engine==1.1.0 --hash=sha256:8290a5b0cb6d670232c30033dda43c43811d305a78cf96df417f62fa646b2635! Will try again.
An error occurred while installing singledispatch==3.4.0.3 --hash=sha256:5b06af87df13818d14f08a028e42f566640aef80805c3b50c5056b086e3c2b9c --hash=sha256:833b46966687b3de7f438c761ac475213e53b306740f1abfaa86e1d1aae56aa8! Will try again.
Anstalling dependencies from Pipfile.lock (443c1c)…
In error occurred while installing tzlocal==2.0.0 --hash=sha256:11c9f16e0a633b4b60e1eede97d8a46340d042e67b670b290ca526576e039048 --hash=sha256:949b9dd5ba4be17190a80c0268167d7e6c92c62b30026cf9764caf3e308e5590! Will try again.
[nstalling initially failed dependencies…
pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/cli/command.py", line 254, in install
[pipenv.exceptions.InstallError]:       editable_packages=state.installstate.editables,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 1874, in do_install
[pipenv.exceptions.InstallError]:       keep_outdated=keep_outdated
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 862, in do_install_dependencies
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, False, failed_deps_queue, retry=False)
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: ['Collecting basemap']
[pipenv.exceptions.InstallError]: ['Killed']
ERROR: ERROR: Package installation failed...
pipenv encountered the following error:
Installing dependencies from Pipfile.lock (443c1c)…
Installing initially failed dependencies…
ERROR    failed to install the pipenv environment
Searching for distribute
Reading https://pypi.org/simple/distribute/
Best match: distribute 0.7.3
Processing distribute-0.7.3-py3.7.egg
distribute 0.7.3 is already the active version in easy-install.pth

Using /usr/local/lib/python3.7/dist-packages/distribute-0.7.3-py3.7.egg
Processing dependencies for distribute
Finished processing dependencies for distribute
INFO: Installing the client desktop application file
INFO: Updating the GTK icon cache
gtk-update-icon-cache: Cache file created successfully.
INFO: Configuring the PostgreSQL server
INFO: PostgreSQL configuration file found at /etc/postgresql/12/main/pg_hba.conf
WARNING: The PostgreSQL king_phisher user already exists! The password will need
WARNING: to be manually updated to match the one automatically generated by this
WARNING: installation script (by using the ALTER ROLE PostgreSQL command) or
WARNING: restored from a previously backed up server configuration file.
INFO: Installing the King Phisher systemd service file in /lib/systemd/system/
INFO: Starting the King Phisher service
--------------------------------------------------------------------------------
```

**Comment by Warafux** [comment]
Same here. Unable to install it. After the script execution a "ModuleNotFoundError: No module named 'gi'" error pops. Installing the module doesn't fix the issue either.

**Comment by zeroSteiner** [comment]
@Warafux That doesn't sound like the same issue. It sounds like you don't have the GObject Introspection libraries installed which should have been handled by the installation script. Can you please open a new ticket and include the output from the installation script?

**Comment by zeroSteiner** [comment]
Well the fix for this in PR #416 has been landed to the dev branch. I have not been able to reproduce this at all so I'm going on faith here that proposed solution will do the trick.

Note that you'll have to check out the dev branch in order to get the change, meaning the original command is still prone to intermittent failures.
